sample-iwa/src/index.js

```diff
@@ -0,0 +1,17 @@
+const extensionId = "gogonhoemckpdpadfnjnpgbjpbjnodgc";
+const request = {
+  type: 'telemetry',
+  telemetry: {
+    infoType: "cpu"
+  }
+}
+
+document.getElementById('hello-btn').addEventListener('click', event => {
+  window.chrome.runtime.sendMessage(
```

Comment:
nit: Maybe put a small comment on what this does and the structure of a request?